src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -173,11 +173,9 @@ protected function writePrompt(OutputInterface $output, Question $question)
     }
 
     /**
-     * @param string $tag
-     *
      * @return string[]
      */
-    protected function formatChoiceQuestionChoices(ChoiceQuestion $question, $tag)
+    protected function formatChoiceQuestionChoices(ChoiceQuestion $question, string $tag)
```

Comment:
that's a BC break for 4.3 - would be OK on master

Reply:
No it's a new method we added 3 hours ago in https://github.com/symfony/symfony/pull/35000.

Reply:
That's still a BC break because #35000 was merged on branch 3.4, so that anyone can now build on it without using type hints.

Reply:
Indeed, my bad 😓